app/views/home/comingsoon.html.haml

```diff
@@ -61,14 +61,12 @@
 
   .home-content
     %h2.orange.center RIT's PREMIERE HACKATHON
```

Comment:
It doesn't look like any styles for content below the fold is working - think it's safe to remove all of this for now. Still works fine as a coming soon splash page